functional/cypress-tests/cypress/support/widgets.js

```diff
@@ -39,10 +39,14 @@ Cypress.Commands.add('confirmAddWidget', () => {
         .click();
 });
 
-Cypress.Commands.add('removeWidget', () => {
-    cy.get('[data-cy="more-menu-button"]')
+Cypress.Commands.add('removeWidget', name => {
+    cy.contains(name)
+        .parents('.MuiCardHeader-root')
+        .find('[data-cy="more-menu-button"]')
```

Comment:
I believe the get that was used before was a little bit better, since it doesn't rely on existence of MuiCardHeader-root class. The best solution here is adding a custom data-cy attribute to whole widget and use it in a similar manner.

Reply:
I agree on that one, there should be data-cy attribute added to whole widget. The reason of modyfying this ine of code was to select only one [data-cy="more-menu-button"] element - the one that belongs to a widget we want to remove.